Explorer/Assets/Scripts/ECS/StreamableLoading/Common/Systems/AssetsLoadingUtility.cs

```diff
@@ -60,7 +46,21 @@ public delegate UniTask<StreamableLoadingResult<TAsset>> InternalFlowDelegate<TA
                         if (intention.CommonArguments.PermittedSources == AssetSource.NONE)
 
                             // conclude now
-                            return new StreamableLoadingResult<TAsset>(unityWebRequestException);
+                            return new StreamableLoadingResult<TAsset>(
+                                reportCategory,
+                                new Exception(
+                                    $"Exception occured on loading {typeof(TAsset)} from {intention.ToString()} with url {intention.CommonArguments.URL}.\n"
+                                    + "No more sources left.",
+                                    unityWebRequestException
+                                )
+                            );
+
+                        // no more sources left
+                        ReportHub.Log(
```

Comment:
I believe we should move this log before `return new...`, otherwise we miss the pot important branch